### ISSUE TITLE
Swig throws in strict mode

### ISSUE THREAD

**Issue by phparkle** [koajs/error#18]
```
$ node --use-strict example.js
/home/parkle/dev/koa-error/node_modules/swig/lib/utils.js:143
        target[key] = obj[key];
                    ^

TypeError: Cannot assign to read only property 'name' of function compiled(locals) {
      var lcls;
      if (locals && contextLength) {
        lcls = utils.extend({}...<omitted>... }
    at Object.exports.extend (/home/parkle/dev/koa-error/node_modules/swig/lib/utils.js:143:21)
    at compile (/home/parkle/dev/koa-error/node_modules/swig/lib/swig.js:622:11)
    at Object.compileFile (/home/parkle/dev/koa-error/node_modules/swig/lib/swig.js:696:17)
    at error (/home/parkle/dev/koa-error/index.js:28:21)
    at Object.<anonymous> (/home/parkle/dev/koa-error/example.js:10:9)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Function.Module.runMain (module.js:467:10)
```

Would be great if koa-error could be used in strict mode, thanks!

**Comment by phparkle** [comment]
https://github.com/paularmstrong/swig/issues/523

**Comment by phparkle** [comment]
Looks like swig is not interested in supporting strict mode... can we maybe use another templating library?

**Comment by coderhaoxin** [comment]
Oh, there are 2 PRs for this

#14 
#13

**Comment by phparkle** [comment]
haha indeed, I forgot to check first, thanks!

**Comment by coderhaoxin** [comment]
Maybe, https://github.com/tj/co-views is considerable

**Comment by coderhaoxin** [comment]
closed by #23
